src/Core/AdminConsole/Repositories/IOrganizationUserRepository.cs

```diff
@@ -27,7 +27,7 @@ public interface IOrganizationUserRepository : IRepository<OrganizationUser, Gui
     /// </summary>
     /// <param name="organizationId">The ID of the organization to get the occupied seat count for.</param>
     /// <returns>The number of occupied seats for the organization.</returns>
-    Task<OrganizationSeatCounts> GetOccupiedSeatCountByOrganizationIdAsync(Guid organizationId);
+    Task<int> GetOccupiedSeatCountByOrganizationIdAsync(Guid organizationId);
```

Comment:
You can delete this method from the code now, it's only the MSSQL sproc that needs to be kept in place for an extra release.

Reply:
removed